cache/remotecache/azblob/exporter.go

```diff
@@ -53,6 +53,10 @@ type exporter struct {
 	config          *Config
 }
 
+func (ce *exporter) Name() string {
+	return "azblobcache"
```

Comment:
We should keep the returning result of `.Name` similar to the existing exporters.

```suggestion
	return "exporting cache to azure blob store"
```

And similar for the other cache exporters.